src/main/kotlin/blackjack/domain/Players.kt

```diff
@@ -0,0 +1,15 @@
+package blackjack.domain
+
+data class Players(val players: List<Player>) {
+    init {
+        require(players.size == PLAYER_COUNT) { "블랙잭 게임에서 플레이어는 2명이어야 합니다" }
```

Comment:
아래와 같이 에러 메시지에도 상수를 사용하면, 추후 값이 변경되더라도 에러 메시지를 추가로 수정하는 공수가 없을 것 같네요. 😃
```suggestion
        require(players.size == PLAYER_COUNT) { "블랙잭 게임에서 플레이어는 ${PLAYER_COUNT}명이어야 합니다" }
```

추가적으로 `PLAYER_COUNT`는 Players 클래스 내부에서만 사용되는 것 같은데, 접근 제한자를 `private`로 제한해주시면 좋을 것 같아요. 😁